web_groupby_expand/README.rst

```diff
@@ -19,7 +19,6 @@ Contributors
 ------------
 
 * Jay Vora <jay.vora@serpentcs.com>
-* Meet Dholakia <m.dholakia.serpentcs@gmail.com>.
```

Comment:
Dont remove. Keep! Keep the greatness afloat.

Reply:
yes, sure sir